src/main/java/uk/gov/ons/ctp/common/rest/RestClient.java

```diff
@@ -165,13 +195,16 @@ public RestTemplate getRestTemplate() {
               + "'";
       if (e.getStatusCode() == HttpStatus.NOT_FOUND) {
         log.warn(errorMessage, e);
```

Comment:
as discussed, please remove the exception parameter for a warning. so no stacktrace is generated

Reply:
Removed